website/client/js/services/questServices.js

```diff
@@ -68,9 +68,10 @@ angular.module('habitrpg')
         }
       }
       text += '---\n\n';
-      text += '**' + window.env.t('rewards') + ':**\n\n';
-      if(quest.drop.items) {
-        for (var item in quest.drop.items) {
+      text += '**' + window.env.t('rewardsAllParticipants') + ':**\n\n';
+      var participantRewards = quest.drop.items ? _.reject(quest.drop.items, 'onlyOwner') : quest.drop.items;
```

Comment:
Also, I believe `_.reject` works with falsy values:

```js
var participantRewards = _.reject(undefined, 'onlyOwner') // []
if (particpantRewards.length > 0) {
  // logic
}
```

Double check that, but it could make the logic more readable.